cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/CradleOperators.java

```diff
@@ -37,18 +37,20 @@
 	private final CassandraDataMapper dataMapper;
 	private final CassandraStorageSettings settings;
 	private final CradleBookOperator cradleBookOp;
+	private final Function<BoundStatementBuilder, BoundStatementBuilder> readAttrs;
 	
-	public CradleOperators(CassandraDataMapper dataMapper, CassandraStorageSettings settings)
+	public CradleOperators(CassandraDataMapper dataMapper, CassandraStorageSettings settings, Function<BoundStatementBuilder, BoundStatementBuilder> readAttrs)
```

Comment:
@Vladimir-Panarin-Exactpro regarding to one of the points in code review, could something like this work instead of passing attributes to each call?

Reply:
Yes, this should be good.